lib/src/test/java/space/taran/arklib/user/score/RootScoreStorageTests.kt

```diff
@@ -0,0 +1,60 @@
+package space.taran.arklib.user.score
+
+import dev.arkbuilders.arklib.user.score.RootScoreStorage
+import dev.arkbuilders.arklib.user.score.ScoreMonoid
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.GlobalScope
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import space.taran.arklib.randomResourceId
+import space.taran.arklib.randomScore
+import java.nio.file.Path
+
+class RootScoreStorageTests {
+
+    private val mScope: CoroutineScope by lazy {
```

Comment:
Remove m prefix everywhere
It is not needed except in the case of backing properties
https://developer.android.com/kotlin/style-guide#naming_2